agent.go

```diff
@@ -656,6 +668,43 @@ func (a *Agent) getBestValidCandidatePair() *CandidatePair {
 }
 
 func (a *Agent) addPair(local, remote Candidate) *CandidatePair {
+	if local.TCPType() == TCPTypeActive && remote.TCPType() == TCPTypePassive {
+		a.log.Debugf("artur, addPair: local %s, remote %s", local, remote)
```

Comment:
Please use a proper format for this and all the following debug messages.

Debug messages should not have this `artur` prefix. And start with a capital letter.

Reply:
This is draft PR, not ready for final review. I am more interested in general about changes.